sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/FluxAutoLockRenewTest.java

```diff
@@ -88,7 +88,7 @@ void setup() {
         lockedUntil = OffsetDateTime.now().plusSeconds(2);
         receivedMessage.setLockToken(LOCK_TOKEN_UUID);
         receivedMessage.setLockedUntil(lockedUntil);
-        renewalFunction = (lockToken) -> Mono.just(OffsetDateTime.now().plusSeconds(1));
+        renewalFunction = (lockToken) -> Mono.just(OffsetDateTime.now().plusSeconds(10));
```

Comment:
If this is a unit test, should this be 10 seconds?

Reply:
I was trying to reduce number of logs which gets printed, every second. When we look in pipeline, we see lot of log messages.